components/alert/alert-toast.js

```diff
@@ -180,6 +182,8 @@ class AlertToast extends LitElement {
 		super.connectedCallback();
 		document.body.addEventListener('d2l-alert-toast-close', this._handleSiblingResize);
 		document.body.addEventListener('d2l-alert-toast-resize', this._handleSiblingResize);
+		document.body.addEventListener('d2l-alert-toast-timer-start', this._handleSiblingCloseTimerStart);
+		document.body.addEventListener('d2l-alert-toast-timer-stop', this._closeTimerStop);
```

Comment:
Let me know if there are potentially better ways of doing this than event listeners. I feel like I just keep piling on the event listeners.

Reply:
Events seem like an excellent choice to me!